### ISSUE TITLE
Language contibution [Portuguese (PT-PT)]

### ISSUE THREAD

**Issue by fabio-silva-pt** [BuIlDaLiBlE/BetterHI3Launcher#29]
[TextStrings_en.cs.txt](https://github.com/BuIlDaLiBlE/BetterHI3Launcher/files/6684995/TextStrings_en.cs.txt)

In line 61 (string "aboutbox_msg") Where it says "Portuguese" it should be "Portuguese pt-br" because thats the brazillian version of portuguese and thats how we usually display it "pt-br" for brazillian and "pt-pt" for native portuguese. So if u add my pt-pt version that should be changed.

Line 82: And in the strings contextmenu_language_(language) it should also say pt-br if ur gonna add the pt-pt version.

Sorry for taking time doing this.

**Comment by BuIlDaLiBlE** [comment]
Yeah, I realize the need to differentiate the two. No problem as I just added French support yesterday!

**Comment by BuIlDaLiBlE** [comment]
So, uh.. for the sake of sanity, can you go over the translation of Brazilian Portuguese I posted in #27 and change it to fit the Portugal Portuguese? I want the text to be more consistent between the two.

**Comment by fabio-silva-pt** [comment]
So, you want me to make the changes i talked about in the Brazilian file? I'm not sure if i understood it correctly.
[TextStrings_pt_pt.cs.txt](https://github.com/BuIlDaLiBlE/BetterHI3Launcher/files/6689990/TextStrings_pt_pt.cs.txt)
[TextStrings_pt_br.cs.txt](https://github.com/BuIlDaLiBlE/BetterHI3Launcher/files/6689991/TextStrings_pt_br.cs.txt)

Ok so, changed both files to have the Portuguese-Brazilian text where it is needed.
Didn't added my language to the credits tho.

![image](https://user-images.githubusercontent.com/79114077/122840168-7cf03380-d2f1-11eb-841d-26e7fceb80ce.png)
Here you might want to change the name and create a new one.

The pt-br file i sent you is the one you sent me edited so its not completed with this stuff for example : 
![image](https://user-images.githubusercontent.com/79114077/122840339-cd679100-d2f1-11eb-8408-20f6cfa2d990.png)

Also, i noticed some probably errors ( nothing very important ) by the Brazilian author version. If he is still in contact i can talk with him or change it myself.

If u are confused with anything and want me to fix something i can do it.
I'm slightly confused with some stuff for example in #5 seeing his updates he didn't updated all the languages but in the text u sent me it was updated. Are you doing that by yourself?

If u want u can send me the full Brazilian file for me to change the necessary stuff there instead of changing the text you sent me.

**Comment by BuIlDaLiBlE** [comment]
> So, you want me to make the changes i talked about in the Brazilian file? I'm not sure if i understood it correctly.
Basically, yeah. If there are any errors, free to fix it in Portugal Portuguese version.
> The pt-br file i sent you is the one you sent me edited so its not completed with this stuff for example :
Yeah, this is not needed for the translation. Honestly, I was thinking of streamlining this process by using something like Mozilla's Pontoon, but couldn't get around to it.
> Also, i noticed some probably errors ( nothing very important ) by the Brazilian author version. If he is still in contact i can talk with him or change it myself.
You can attach the text here with only those lines that need to be changed, I'll update it.
> I'm slightly confused with some stuff for example in #5 seeing his updates he didn't updated all the languages but in the text u sent me it was updated. Are you doing that by yourself?
Actually, yeah. I add simple phrases that I'm confident are not wrong myself. If there is still something, feel free to fix! It was purely due to me not wanting to bother the translators too much (especially with small lines), actually.
> If u want u can send me the full Brazilian file for me to change the necessary stuff there instead of changing the text you sent me.
What I've sent to this thread is the latest version of Brazil Portuguese at the moment. I don't host translation files on GitHub anymore due to the new translation download feature that allows me to update translations without making a new update for the launcher.

**Comment by fabio-silva-pt** [comment]
### About the Brazilian file:  

> "aboutbox_msg": "Bem, é muito mais avançado, não é? :^)\nEste projeto foi feito com a esperança de que muitos capitães tivessem uma melhor experiência com o jogo.\nEle não é afiliado à miHoYo e é totalmente código-aberto.\nQualquer comentário será muito bem vindo.\nAgradecimentos especiais à esses contribuidores do GitHub:\nSinsOfSeven - Contribuição de resolução personalizada\nProxy-E23 - Tradução da idioma espanhola\nSpookyKisuy - Tradução da idioma portuguêsa-brasileira\nbulawin1, Vrontis - Tradução da idioma alemã\nKorewaLidesu - Tradução da idioma vietnamita\nSm1leAway - Tradução da idioma sérvia\nVoRTeXStarz - Tradução da idioma tailandesa",

Maybe change to 

> "aboutbox_msg": "Bem, é muito mais avançado, não é? :^)\nEste projeto foi feito com a esperança de que muitos capitães tivessem uma melhor experiência com o jogo.\nEle não é afiliado à miHoYo e é totalmente código-aberto.\nQualquer comentário será muito bem vindo.\nAgradecimentos especiais à esses contribuidores do GitHub:\nSinsOfSeven - Contribuição de resolução personalizada\nProxy-E23 - Tradução do idioma espanhol\nSpookyKisuy - Tradução do idioma português-brasileiro\nbulawin1, Vrontis - Tradução do idioma alemão\nKorewaLidesu - Tradução do idiomo vietnamita\nSm1leAway - Tradução do idioma sérvio\nVoRTeXStarz - Tradução do idioma tailandes",

**Comment by SpookyKisuy** [comment]
Yo Fábio could we talk on Discord? My username is spx#0727

**Comment by BuIlDaLiBlE** [comment]
A few new strings!~
```
"msgbox_feature_not_available_msg": "This feature is not available for selected server.",
"msgbox_install_1_msg": "Select where would you like to install the game.",
"msgbox_move_1_msg": "Select where would you like to move the game.",
```
Also a part of `downloadcachebox_msg` was changed as well, I need this translated:
```
"Full cache" contains the entire cache, including event pictures and settings. "Numeric files" contains settings only, those files are updated most frequently.
```

**Comment by fabio-silva-pt** [comment]
"msgbox_feature_not_available_msg": "Esta funcionalidade não está disponível no servidor selecionado.", 
"msgbox_install_1_msg": "Selecione onde gostaria de instalar o jogo.", 
"msgbox_move_1_msg": "Selecione onde gostaria de mover o jogo.", 

"Cache inteiro" contém todo o cache, incluindo imagens de eventos e configurações. "Ficheiros numéricos" contém apenas as configurações, esses ficheiros são atualizados com mais frequência.

**Comment by fabio-silva-pt** [comment]
"contextmenu_custom_launch_options": "Escolher opções de lançamento customizadas", 
"msgbox_custom_launch_options_msg": "Usuários avançados podem costumizar as opções de lançamento aqui.\nNota: Elas são aplicadas independentemente por servidor.",

**Comment by BuIlDaLiBlE** [comment]
In need of a translation again!
`The entire game cache will be checked and can be downloaded if needed. It can take a while.\nHi3Mirror is a project courtesy of {0}.\nContinue?`

**Comment by fabio-silva-pt** [comment]
`O cache inteiro será verificado e pode ser instalado se necessário. Pode demorar um bocado.\nHi3Mirror é um projeto de cortesia de {0}. \nContinuar?`

**Comment by BuIlDaLiBlE** [comment]
A couple of new ones.
```
"label_merging": "Merging",
"label_merged": "Merged {0}%",
"label_merge_speed": "Merge speed:",
"announcementbox_do_not_show": "Do not show me this next time",
```

**Comment by fabio-silva-pt** [comment]
```
"label_merged": "Misturado {0}%",
"label_merge_speed": "Velocidade de mistura:",
"announcementbox_do_not_show": "Não me mostre isto da próxima vez", 
```

**Comment by BuIlDaLiBlE** [comment]
I have changed `msgbox_download_type_1_msg` from
```
This will reset game's resource download type in an attempt to fix the infamous update loop which does not let you enter the game.\nContinue?
```
to
```
This will reset game's resource download type and allow you to reselect which assets to download.\nContinue?
```
Can you please retranslate that?

**Comment by BuIlDaLiBlE** [comment]
Two new strings:
```
"contextmenu_open_screenshots_dir": "Open screenshots directory"
"msgbox_no_screenshot_dir_msg": "There are no screenshots. Go take some!"
```
